wellregistry/registry/models.py

```diff
@@ -6,6 +6,20 @@
 from django.core.validators import MaxValueValidator, MinValueValidator
 
 
+class CountryLookup(models.Model):
+    """
+    Country lookup table for the Registry app.
+
+    Used to populate drop downs.
+
+    """
+    country_code = models.CharField(max_length=10)
```

Comment:
So this creates an artificial key. I think we should we be using the country_code? I think this is what the legacy NGWMN expects.

Reply:
Sounds good. I'll make the changes.